test/katex-spec.js

```diff
@@ -1382,6 +1382,15 @@ describe("An HTML font tree-builder", function() {
         span = "<span class=\"mord mathbb\" style=\"color:blue;\">R</span>";
         expect(markup).toContain(span);
     });
+
+    it("should throw TypeError when the expression is of the wrong type", function() {
+        expect(function() {
+            katex.renderToString({badInputType: "yes"});
+        }).toThrowError(TypeError);
+        expect(function() {
+            katex.renderToString(["foo", {badInputType: "yes"}]);
+        }).toThrowError(TypeError);
+    });
```

Comment:
Suggestion of testing for more diverse types done in https://github.com/Khan/KaTeX/pull/540/commits/95acda3824e6587160554e33ea0674aaaf63f017